### DISCUSSION TITLE
Job fails without errors - possible timeout

### DISCUSSION THREAD

**Discussion by yuvalhnoga** [renovatebot/renovate#30284] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have Renovate running with the Mend GitHub app in my organization. For one of my repositories, the job fails most of the time without any errors in the logs.

The last entries in the logs are:
```
DEBUG: hostRules: applying Basic authentication for hub.docker.com
DEBUG: Found no satisfying versions with 'semver-coerced' versioning
```

Usually, after this point, the PRs are created. However, it seems that the process fails here. From my observations, when the job succeeds, it completes in less than 30 minutes. When it fails, it does so exactly after 30 minutes. The job duration shows `-` when it fails.

I would like to know how to debug this issue and if there is some sort of timeout implemented for jobs running in Mend. This repository is a monorepo containing many Terraform configuration files. It includes around 6500 terragrunt.hcl files, which look for dependencies in another GitHub repository based on tags from that repository.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by nabeelsaabna** [comment]
@yuvalhnoga your assumption is correct, each run is limited to 30 minutes

**Reply by yuvalhnoga** [reply]
@nabeelsaabna What do you suggest to do in this case? I am trying to figure out why the run takes that long and if I can do anything about it and if not then is it possible to increase the timeout?

**Reply by nabeelsaabna** [reply, marked as answer]
As a first step, it would be helpful if you can run manually on this repository and see how long it takes to finish a single run.
follow [self-hosting.md](https://github.com/renovatebot/renovate/blob/main/docs/usage/examples/self-hosting.md) 

At the moment increasing the timeout is not an available option

**Reply by yuvalhnoga** [reply]
It does seem to work locally but takes around 30 minutes. I will see how I can maybe make it faster and if not I will use the self hosted github action instead for this specific repo. Thanks for the help!
